docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -394,6 +394,11 @@ <h3 id="known_issues_5.3.1">Known Issues</h3>
           <td>Portable Globe Cutter fails in the last step when using HTTPS</td>
           <td>Switch to <code>http</code>. </td>
         </tr>
+        <tr>
+          <td>1381</td>
+          <td>Stopping Fusion results in double free or corruption error in system manager</td>
+          <td>No current work around.</td>
```

Comment:
Do we want to mention that it doesn't seem to cause any issues?

Reply:
I just updated the doc to include a mention about it.